documentcloud/projects/tests/test_views.py

```diff
@@ -117,6 +117,70 @@ def test_destroy(self, client, project):
         assert not Project.objects.filter(pk=project.pk).exists()
 
 
+@pytest.mark.django_db()
+class TestProjectFilters:
+    def test_filter_is_shared(self, client):
+        """Test filtering projects that are shared with the user"""
+        # Create users
+        owner = UserFactory()
+        collaborator = UserFactory()
+        non_collaborator = UserFactory()
+
+        # Create projects
+        shared_project = ProjectFactory(user=owner)
+        Collaboration.objects.create(
+            project=shared_project,
+            user=collaborator,
+            creator=owner,
+            access=CollaboratorAccess.admin,
+        )
+
+        owned_only_project = ProjectFactory(user=owner)
+
+        # Create a project that the user does not collaborate on
+        ProjectFactory(user=non_collaborator)
+
+        # Authenticate as the collaborator
+        client.force_authenticate(user=collaborator)
+
+        # Filter for shared projects
+        response = client.get("/api/projects/", {"is_shared": True})
+        assert response.status_code == status.HTTP_200_OK
+        response_json = json.loads(response.content)
+
+        # Check that only the shared project is included
+        assert len(response_json["results"]) == 1
+        assert any(p["id"] == shared_project.id for p in response_json["results"])
```

Comment:
Since you already asserted there is exactly one project, I think checking `response_json["results"][0]` is clearer than using `any`.
Similarly, if you assert the only project has the correct ID, you can skip the next assert that it does not have the wrong ID.